Wise_Game/Wise_Game/main.swift

```diff
@@ -0,0 +1,344 @@
+//
+//  main.swift
+//  TextAdventure
+//
+//  Created by Amy Alsaydi on 10/1/19.
+//  Copyright © 2019 Amy Alsaydi. All rights reserved.
+//
+
+/*
+ Assignment Requirements:
+ - no compiler errors/warnings ✅
+ - 10 user decisions✅
+ - user enter number ✅
+ - switch statement ✅
+ - if/else statment ✅
+ - type annotation ✅
+ - type inference ✅
+ - 1 tuple ✅
+ - 1 range ✅
+ - for loop ✅
+ - While loop ✅
+ 
+ */
+
+import Foundation
+
+print("WAKE UP YOU GOT US INTO THIS MESS!!!!")
+sleep(3)
+print("Some one just slid a note ✉️ under the door, should I open it?")
+let answer = readLine()?.lowercased()
```

Comment:
Good job lowercasing the user input so it would always lowercase their response to minimize error. To also minimize further error on the user side you would want to unwrap their response. It's optional and can fatally crash your app every time. You can use nil coalescing      readLine()?.lowercased() ?? "no"  also you don't want your default to be blank you wnat to be able to use it in your code when the time comes